### PR TITLE
#5 in contest page

### PR DESCRIPTION
**Pull request by dhanzy**
### What this PR does (required):
- Contest submits data to database through the contest page

### Screenshots / Videos (required):
- 
Screenshot (24)

### Any information needed to test this feature (required):
- Sign in. Click on "create contest" on the header. Then create a new contest.

### CODE REVIEW

**Comment by dhanzy** [comment]
@rajivtitus i have done the integration of the contest page. Have a look at it

File: client/src/pages/Contest/ContestForm/ContestForm.tsx [hunk 1]
```diff
@@ -37,20 +37,58 @@ interface ContestFormValues {
...
+
+export const ContestForm: React.FC<Props> = ({ handleSubmit }) => {
   const classes = useStyles();
   const [selectedImages, setSelectedImages] = useState<string[]>([]);
   const [selectedDateTime, handleDateTimeChange] = useState<MaterialUiPickersDate | Moment>(moment());
   const [timeZone, setTimeZone] = useState<string | unknown>(moment.tz.guess());
 
   const selectImages = (image: string) => {
     let newSelectedImages = selectedImages;
+    console.log(image);
```
Comment (rajivtitus) [inline client/src/pages/Contest/ContestForm/ContestForm.tsx]: Please remove

File: client/src/pages/Contest/ContestForm/ContestForm.tsx [hunk 2]
```diff
@@ -37,20 +37,58 @@ interface ContestFormValues {
...
     if (newSelectedImages.includes(image)) {
       newSelectedImages = newSelectedImages.filter((img) => img !== image);
     } else {
       newSelectedImages = [...newSelectedImages, image];
     }
+    console.log(`SelectedImages ${selectedImages}`);
```
Comment (rajivtitus) [inline client/src/pages/Contest/ContestForm/ContestForm.tsx]: Please remove console.logs throughout

File: client/src/pages/Contest/Contest.tsx [hunk 3]
```diff
@@ -1,13 +1,65 @@
...
+      date: MaterialUiPickersDate | Moment | string | null;
+      time: MaterialUiPickersDate | Moment | string | null;
+      timeZone: string;
+      imageFiles: string[];
+    }>,
+  ) => {
+    createContest(title, description, prizeAmount, createDeadlineDate(date, time, timeZone), imageFiles).then(
+      (data) => {
+        if (data.error) {
+          setSubmitting(false);
```
Comment (rajivtitus) [inline client/src/pages/Contest/Contest.tsx:54]: Maybe update the snackbar here on success/error?

File: client/src/pages/Contest/Contest.tsx [hunk 4]
```diff
@@ -1,13 +1,65 @@
...
+        } else if (data.success) {
+          return history.goBack();
```
Comment (rajivtitus) [inline client/src/pages/Contest/Contest.tsx]: Than going to the previous page maybe we can reroute to the discover page? (Where all our contests reside)

**Review by rajivtitus — APPROVED** [review]
LGTM!

Just a few comments, please address before merging

**Comment by dhanzy** [comment]
Okay @rajivtitus Workin on it now.

**Comment by dhanzy** [comment]
@rajivtitus  Resolved all the neccessary comments. Thanks

**Comment by HAZberi** [comment]
LGTM!

**Review by HAZberi — APPROVED** [review]
Just had a quick glance over. Looks good. Thank you Daniel.

**Comment by dhanzy** [comment]
Thanks @HAZberi

**Pull request merged**